model/src/id/mod.rs

```diff
@@ -281,6 +299,18 @@ impl<T> FromStr for Id<T> {
     }
 }
 
+impl<T> From<Id<T>> for u64 {
```

Comment:
these two should be moved above the `impl<T> From<NonZeroU64>` implementation, for alphabetical order

Reply:
Thanks, wasn't sure what's the proper ordering here, so I just put it _somewhere_ for now. Will update this is now, just a sec.